README.md

```diff
@@ -31,72 +31,582 @@ _Kev_ reduces the need for Kubernetes expertise in the team. The generated Kuber
 
-### Initiate a project
+## Quickstart
 
-Run the following command from within your project directory to initialise Kev project:
+- `kev init` - identifies a project's Compose Kubernetes source files and creates Compose environment overrides.
+- `kev render` - detects, applies any config changes and generates deployment manifests.
+- `kev help` - run it if you're a little lost.
+
+### init
+
+Run the following command within your project directory:
 
 ```sh
 kev init
 ```
 
-The command above will auto-detect default `docker-compose.yaml` and `docker-compose.override.yaml` files (if present) in the project directory, and tracks them as Kubernetes deployment sources.
+This identifies the default `docker-compose.yaml` and (if present) `docker-compose.override.yaml` files in your project directory. They will be used as Compose Kubernetes sources.
 
-If you want to point _Kev_ at set of alternative compose files, simply pass them in with `-f` flag. Multiple source compose files can be specified by providing `-f` flag multiple times.
+Also, it creates a default `dev` environment and its Compose override file.  
 
-And, to individually control the configuration of your project for deployments in separate target environments, just specify their respective names with `-e` flag, e.g.:
+Here's another example. It uses an alternate `docker-compose` file with `stage` + `prod` environments:
 
 ```sh
 kev init -f my-docker-compose.yaml -e stage -e prod
 ```
 
-If no environment has been specified, a default one will be named `dev`.
+It makes use of,
+- `-f` flag, to specify an alternate filename.
+- `-e` flags, to specify different deployment environments.
 
-Once the project has bootstrapped you should see a few new files added to your project tree, similar to the list below:
+Creating the files below in your project directory:
 
 ```sh
-├── docker-compose.kev.stage.yaml # stage environment override
-├── docker-compose.kev.prod.yaml  # prod environment override
-├── kev.yaml                      # kev project manifest
+├── docker-compose.kev.stage.yaml       # stage Compose environment override file
+├── docker-compose.kev.prod.yaml        # prod Compose environment override file
+├── kev.yaml                            # kev project manifest
 ├── ...
 ```
 
-Your project is now ready and environment specific configuration files generated. _Kev_ will infer useful configuration details already present in the source compose file(s). If this is not possible it'll assume sensible defaults. Users have an easy way to manually alter values directly in the environment specific override file as and when required.
+Here's what happened, _Kev_,
+- Inferred the configuration details already present in your compose Kubernetes deployment sources.
+- Assigned sensible defaults for any config it couldn't infer.
+- Created Compose overrides files for the `stage` and `prod` environments.
+
+That's it, _Kev_ is now bootstrapped and ready!
```

Comment:
`That's it, _Kev_ is now bootstrapped and ready!` -> `That's it, your _Kev_ project is now bootstrapped and ready!`